elasticsearch/templates/statefulset.yaml

```diff
@@ -94,14 +94,16 @@ spec:
           privileged: true
         image: "{{ .Values.image }}:{{ .Values.imageTag }}"
         command: ["sysctl", "-w", "vm.max_map_count={{ .Values.sysctlVmMaxMapCount}}"]
+        resources:
```

Comment:
Can you add a [small test](https://github.com/elastic/helm-charts/blob/master/elasticsearch/tests/elasticsearch_test.py) to prove that this works and that the indentation is correct. The development environment is documented in the [testing](https://github.com/elastic/helm-charts/tree/master/elasticsearch#testing) section of the readme.

Reply:
Ok I will see if I can work out how that works.

Reply:
My latest commit should address that.
I ran the test suite locally and got a pass so I think I did it right.
Let me know if I need to add more tests.

Reply:
Looks perfect! And a single test is more than enough for this since it is just passing through the yaml. Just want something to make sure the indentation is good and will remain that way if it ever gets refactored in the future 👍